### DISCUSSION TITLE
Questions about Prometheus binary files

### DISCUSSION THREAD

**Discussion by FSJ-047** [prometheus/prometheus#12883] [General]
The binary file of Prometheus contains "BEGIN RSA Testing KEY". May I ask what this is used for

**Comment by voltagex** [comment]
a quick Google shows this is part of a test file. https://github.com/golang/go/blob/master/src/crypto/x509/pem_decrypt_test.go
Unsure why it gets compiled into Prometheus

**Comment by FSJ-047** [comment]
I found out that he is a test certificate in Golang's HTTP library, but I am not sure what the purpose of compiling him into a binary file is.
